.github/workflows/publish_docker_images.yml

```
@@ -79,7 +79,7 @@ jobs:
           touch "/tmp/digests/${digest#sha256:}"
 
       - name: Upload digest
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4.4.0
         with:
           name: digest
```

looks like we hit the breaking change you mentioned 
```suggestion
          name: digest-${{ matrix.platform }}
```
and line 96
```diff
-          name: digest
+          name: digest-*
```